src/css/dropdown-menu.scss

```diff
@@ -1,41 +1,42 @@
-// *** DROPDOWN MENU STYLES ***
+/* *** DROPDOWN MENU STYLES *** */
 .dropdown {
-  background-color: var(--section-bcg-color);
-  border: 1px solid var(--base-color);
-  border-radius: 20px;
+  background-color: $section-bcg-color;
+  border: 1px solid $base-color;
+  border-radius: 1.4em;
   grid-area: filter;
-  min-width: 120px;
+  min-width: 7.5em;
   position: absolute;
-  top: 72px;
+  top: 4.3em;
   width: fit-content;
 }
 
 .dropdown {
+  p {
+    &:hover {
+      color: $accent-color;
+    }
+  }
   &>header {
-    border-radius: 20px;
+    border-radius: 2em;
     cursor: pointer;
     margin: 0;
-    padding: 6px;
+    padding: 0.2em;
     &>p {
-      color: var(--base-color);
-      font-size: 16px;
+      color: $base-color;
+      font-size: 1.1em;
```

Comment:
@RayRedGoose just going to ask for a few clarifications,  when you use the `&>header` and `&>p` you are selecting the elements within the drop down?